packages/gallery/src/components/gallery/galleryRendererTypes.ts

```diff
@@ -0,0 +1,7 @@
+export interface GalleryRendererProps {
+  container: any;
```

Comment:
@ritabazak Please place all files that contains interfaces only (not logic) inside the `common/interfaces` folder

Reply:
In this PR? or another PR?

Reply:
I don't see any value in creating a separate PR